tests/test_log_api.py

```diff
@@ -0,0 +1,172 @@
+import json
+
+import httpretty
+import pytest
+from mock import patch
+from mock import MagicMock
+
+from lecli import log_api
+from examples import misc_examples as misc_ex
+from examples import response_examples as resp_ex
+
+@httpretty.activate
+@patch('lecli.api_utils.get_ro_apikey')
+@patch('lecli.log_api._url')
+def test_get_logs(mocked_url, mocked_ro_apikey, capsys):
+    mocked_url.return_value = misc_ex.MOCK_LOGAPI_URL
+    mocked_ro_apikey.return_value = misc_ex.TEST_APIKEY_WITH_VALID_LENGTH
+    httpretty.register_uri(httpretty.GET, misc_ex.MOCK_LOGAPI_URL,
+                           status=200,
+                           content_type='application/json',
+                           body=json.dumps({'logs': resp_ex.log_response}))
+
+    log_api.get_logs()
+    out, err = capsys.readouterr()
+
+    assert '"id": "12345"' in out
+
+@httpretty.activate
+@patch('lecli.api_utils.get_ro_apikey')
+@patch('lecli.log_api._url')
+def test_get_log(mocked_url, mocked_ro_apikey, capsys):
+    log_id = misc_ex.TEST_LOG_RESOURCE_ID
+    mocked_url.return_value = misc_ex.MOCK_LOGAPI_URL
+    mocked_ro_apikey.return_value = misc_ex.TEST_APIKEY_WITH_VALID_LENGTH
+    httpretty.register_uri(httpretty.GET, misc_ex.MOCK_LOGAPI_URL + '/' + log_id,
+                           status=200,
+                           content_type='application/json',
+                           body=json.dumps({'log': resp_ex.log_response}))
+
+    log_api.get_log(log_id)
+    out, err = capsys.readouterr()
+
+    assert '"id": "12345"' in out
+
+
+@httpretty.activate
+@patch('lecli.api_utils.get_rw_apikey')
+@patch('lecli.log_api._url')
+def test_create_log_with_default_source_type(mocked_url, mocked_rw_apikey, capsys):
+    mocked_url.return_value = misc_ex.MOCK_LOGAPI_URL
+    mocked_rw_apikey.return_value = misc_ex.TEST_APIKEY_WITH_VALID_LENGTH
+    httpretty.register_uri(httpretty.POST, misc_ex.MOCK_LOGAPI_URL,
+                           status=201,
+                           body='{"log": {"name": "test log","id": "2caec19c-d8a2-40ef-9c1e-91e89157fe28",'
```

Comment:
I think here you can reuse the response body in https://github.com/rapid7/lecli/pull/32/files#diff-68084ff0bea49386c8ff8d6bd61ce0e9R143